### PR TITLE
ecm-tools: init at github master

### PR DESCRIPTION
**Pull request by m3tti**
###### Things done

- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [x] Linux
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/misc/ecm-tools/default.nix [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchgit }:
+stdenv.mkDerivation rec {
+  name = "ecm-tools";
+  version = "master";
```
Comment (FRidh) [inline pkgs/applications/misc/ecm-tools/default.nix]: The name should consists of a name part and a version part. Please see the Nixpkgs manual on the naming guidelines.

Comment (m3tti) [inline pkgs/applications/misc/ecm-tools/default.nix]: hi the current maintainer doesn't have any version number. should i just go for 1.0?

Comment (joachifm) [inline pkgs/applications/misc/ecm-tools/default.nix]: See https://github.com/alucryd/ecm-tools/releases/tag/v1.0.3

**Comment by matthewbauer** [comment]
@GrahamcOfBorg build ecm-tools

**Review by matthewbauer — CHANGES_REQUESTED** [review]
Need release tag

**Comment by c0bw3b** [comment]
Stalled for more than a year.

**Pull request closed without merging**